spec/unit/facts/network_nexthop_ip_spec.rb

```diff
@@ -8,6 +8,7 @@
   end
   context 'on a Linux host' do
     before do
+      Facter.fact(:virtual).stubs(:value).returns('physical')
```

Comment:
Odd... I don't understand.  What error do you get otherwise?  I can't reproduce so far.